clickable

```diff
@@ -1093,7 +1099,35 @@ class PythonClickable(PureClickable):
     # The only difference between this and the Pure template is that this doesn't force the "all" arch
     pass
 
+class GoClickable(Clickable):
+    def go_install(self):
+        get_output=True
+        if not os.path.exists(self.temp):
+            os.makedirs(self.temp)
```

Comment:
This should already exist, not need to do it again here.

Reply:
done